lib/improver/spotdata/tests/test_common_functions.py

```diff
@@ -0,0 +1,605 @@
+# -*- coding: utf-8 -*-
```

Comment:
That's because you are looking at the class names, which as per the style guide, should be written as they are. The individual tests are titled test_...